### DISCUSSION TITLE
Websites Protected with cloudflare can katana bypass it to crawl?

### DISCUSSION THREAD

**Discussion by Greatz08** [projectdiscovery/katana#828] [General]
Generally cloudflare detects bot and similar activities so can this crawler crawl easily without any errors through cloudflare advanced protected websites? If yes then how and if no then what he done to solve this

**Comment by Mzack9999** [comment]
There isn't an all in one solution for this issue. If the website is protected by cloudflare generally you should keep a low parallelism and you might get better chances with headless crawling that uses native chrome, so more difficult to fingerprint.
